### PR TITLE
Include Scoring description and fix error

### LINKED ISSUE #107: KYC Match Readiness Checklist
Include the checklist for meta-release Fall24, including documentation, ATPs and alignment with commonalities (0.4.0)

### PR DESCRIPTION
**Pull request by jgarciahospital**
#### What type of PR is this?

Add one of the following kinds:

* correction
* enhancement/feature
* documentation

#### What this PR does / why we need it:

Including:
- Clarification on scoring distance algorithm
- Aligning versioning (RC) with commonalities https://github.com/camaraproject/Commonalities/blob/r0.4.0-alpha.1/documentation/Camara_Versioning_Guidelines.md
- Align Info and Server objects with commonalities https://github.com/camaraproject/Commonalities/blob/r0.4.0-alpha.1/documentation/API-design-guidelines.md
- Open to RC creation

#### Which issue(s) this PR fixes:

Fixes #107

### CODE REVIEW

**Comment by jgarciahospital** [comment]
@rartych please review the `info` and `servers` objects and error management, to confirm if already aligned with commonalities 0.4

**Review by GillesInnov35 — DISMISSED** [review]
Ok thanks, compliant with API version in URL recommendations

**Review by fernandopradocabrillo — DISMISSED** [review]
LGTM

**Review by HuubAppelboom — COMMENTED** [review]
I think this should be a float.  With an integer you loose too much unnecessary precision with Jaro-Winkler. Typically people will already classify something as a mismacth when a score reaches 90-95%

**Comment by hdamker** [comment]
@jgarciahospital is this PR already the release PR?

If yes, it should contain also the update of the changelog, readiness checklist, etc - for both API definitions in the repository (KYC Fill-in is still in "wip").

If the PR is meant just to do the last changes *before* the release PR, then it should not write already the version for the release - that will be done in the release PR.

File: code/API_definitions/kyc-match.yaml [hunk 1]
```diff
@@ -51,19 +52,16 @@ info:
...
-  version: wip
-
-
+  version: 0.2.0-rc.1
+externalDocs:
+  description: Product documentation at Camara.
+  url: https://github.com/camaraproject/
 servers:
-  - url: '{apiRoot}/{basePath}'
+  - url: '{apiRoot}/kyc-match/v0.2'
```
Comment (hdamker) [inline code/API_definitions/kyc-match.yaml:60]: That would be (for the release candidate):
```suggestion
  - url: '{apiRoot}/kyc-match/v0.2rc1'

```

**Comment by jgarciahospital** [comment]
@ToshiWakayama-KDDI we have removed the RC creation since it requires also to include the KYC fill-in code, so let's focus this PR in aligning the API code close together with #115 , please merge both as already reviewed and we can then create the proper RC ones.

**Comment by ToshiWakayama-KDDI** [comment]
Hi @jgarciahospital ,

Thanks.  But we have decided to postpone PR #115 till Spring25, havent't we?

BR
Toshi

**Comment by jgarciahospital** [comment]
You are right, sorry, I was thinking on #114

**Comment by ToshiWakayama-KDDI** [comment]
> You are right, sorry, I was thinking on #114

Hi @jgarciahospital ,

OK, PR #114, not PR #115.  That makes sense.  Thanks.

BR
Toshi

**Comment by ToshiWakayama-KDDI** [comment]
Hi @jgarciahospital , @fernandopradocabrillo , @GillesInnov35 ,

Looks good to me, but I see the message 'this branch has conflicts that must be resolved'.  Could you check it, @jgarciahospital , @fernandopradocabrillo , please?

Thanks,
Toshi

**Comment by jgarciahospital** [comment]
> Hi @jgarciahospital , @fernandopradocabrillo , @GillesInnov35 ,
> 
> Looks good to me, but I see the message 'this branch has conflicts that must be resolved'. Could you check it, @jgarciahospital , @fernandopradocabrillo , please?
> 
> Thanks, Toshi

Now solved.

**Review by ToshiWakayama-KDDI — APPROVED** [review]
LGTM

**Comment by ToshiWakayama-KDDI** [comment]
Hi @jgarciahospital , 

Thanks very much.   I have just approved this.

BR
Toshi

**Review by fernandopradocabrillo — APPROVED** [review]

**Comment by ToshiWakayama-KDDI** [comment]
Thanks, @fernandopradocabrillo.

Since Gills is on holiday and this PR has to be merged immediately for Fall24 Meta-release, I will merge this right now without Gilles' approval.

Thanks,
Toshi

**Pull request merged**